packages/salesforcedx-vscode-apex/src/commands/forceGenerateFauxClasses.ts

```diff
@@ -112,18 +111,30 @@ export class ForceGenerateFauxClassesExecutor extends SfdxCommandletExecutor<{}>
 
     taskViewService.addCommandExecution(execution, cancellationTokenSource);
 
-    const projectPath: string = getRootWorkspacePath();
     const gen: FauxClassGenerator = new FauxClassGenerator(
       execution.cmdEmitter,
       cancellationToken
     );
 
+    const commandName = execution.command.logName;
     try {
-      const result = await gen.generate(projectPath, SObjectCategory.ALL);
-      console.log('Generate success ' + result);
-    } catch (e) {
-      console.log('Generate error ' + e);
+      const result = await gen.generate(
+        vscode.workspace.workspaceFolders![0].uri.fsPath,
+        SObjectCategory.ALL,
+        refreshSource
+      );
+      console.log('Generate success ' + result.data);
+      this.logMetric(commandName, startTime, result.data);
+    } catch (result) {
+      console.log('Generate error ' + result.error);
```

Comment:
Now log errors and successes with the number of objects fetched - if the command got far enough to collect that information.